qemu/tests/cfg/live_snapshot_chain.cfg

```diff
@@ -24,8 +24,23 @@
             check_base_image_image1 = yes
             Ubuntu:
                 post_snapshot_cmd = {shell:apt-get update -y}
+        - check:
+            snapshot_chain = "image1 sn1 sn2"
+            image_name_sn1 = "images/sn1"
+            image_name_sn2 = "images/sn2"
+            file_create = yes
+            check_base_image = yes
+        - oops:
+            post_snapshot_cmd = {shell_no_reply:echo c > /proc/sysrq-trigger}
+            check_base_image = yes
+            operate_timeout = 10
+            check_alive_cmd = ""
         - long_chain:
-            snapshot_chain = "image1 sn1 sn2 sn3 sn4 sn5 sn6 sn7 sn8 sn9 sn10 sn11 sn12 sn13 sn14 sn15 sn16 sn17 sn18 sn19 sn20"
+            need_reboot = yes
+            file_create = yes
+            check_base_image_image1 = yes
+            check_base_image_sn64 = yes
+            snapshot_chain = "image1 sn1 sn2 sn3 sn4 sn5 sn6 sn7 sn8 sn9 sn10 sn11 sn12 sn13 sn14 sn15 sn16 sn17 sn18 sn19 sn20 sn21 sn22 sn23 sn24 sn25 sn26 sn27 sn28 sn29 sn30 sn31 sn32 sn33 sn34 sn35 sn36 sn37 sn38 sn39 sn40 sn41 sn42 sn43 sn44 sn45 sn46 sn47 sn48 sn49 sn50 sn51 sn52 sn53 sn54 sn55 sn56 sn57 sn58 sn59 sn60 sn61 sn62 sn63 sn64"
```

Comment:
Can we just set a snapshot number here for the chain and generate this list in the code accordingly?